src/components/grapher/GrapherTool.vue

```diff
@@ -25,6 +44,32 @@ export default Vue.extend({
     grapherToolDots(): StuntSheetDot[] {
       return this.$store.state.grapherToolDots;
     },
+    showDotLabels(): boolean {
+      return this.$store.state.showDotLabels;
+    },
+    grapherToolDotLabels(): [StuntSheetDot, string][] {
+      const dotLabels = this.$store.getters.getDotLabels;
+      const dots: StuntSheetDot[] = this.grapherToolDots;
+      return dots.map((dot) => {
+        return [
+          dot,
+          dotLabels !== null &&
+          dot.dotLabelIndex !== null &&
+          dot.dotLabelIndex < dotLabels.length
+            ? dotLabels[dot.dotLabelIndex]
+            : "",
+        ];
+      });
+    },
+    showSelectionLasso(): boolean {
+      return this.$store.state.showSelectionLasso;
+    },
+    selectionLasso(): string {
+      const lassoPointStrings: string[] = this.$store.state.selectionLasso.map(
+        (point: [number, number]) => point.join(",")
+      );
```

Comment:
nit: Can we leave a comment here to say that this matches the format for `points` in the html element `polyline`? You can include the MDN doc link as well if you want https://developer.mozilla.org/en-US/docs/Web/SVG/Element/polyline

Reply:
done!